components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Engine/DataFlowProcessingEngine.cs

```diff
@@ -388,6 +389,8 @@ public class DataFlowProcessingEngine : IProcessingEngine, IDisposable {
         /// </summary>
         private readonly int _maxOutgressMessages;
 
+        private readonly bool _hasRoutingInfo;
```

Comment:
`_hasRoutingInfo` does not seem to be used. Can we remove it ?

Reply:
sure, done